src/routes/SelectRepos/index.js

```diff
@@ -82,6 +82,20 @@ export class SelectRepos extends React.Component {
           Can&#8217;t find one of your repos here? Make sure you are watching it
           on Github!
         </p>
+        {this.props.selectedRepos.length ? (
+          <p className={theme.intro}>
+            <Button
```

Comment:
At the moment when a repo is first selected they all shift down as the button is rendered into the page. I think it would be a better experience if there was some empty space above the selection list which the button could appear in so that it doesn't cause other elements to shift down when it appears.

Reply:
😕